config/0000_00_cluster-version-operator_01_clusteroperator.crd.yaml

```diff
@@ -0,0 +1,80 @@
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: clusteroperators.config.openshift.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.version
+    description: The version the operator is at.
+    name: Version
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Available")].status
+    description: Whether the operator is running and stable.
+    name: Available
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Progressing")].status
+    description: Whether the operator is processing changes.
+    name: Progressing
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Failing")].status
+    description: Whether the operator is failing changes.
+    name: Failing
+    type: string
+  - JSONPath: .status.conditions[?(@.type=="Available")].lastTransitionTime
+    description: The time the operator's Available status last changed.
+    name: Since
+    type: date
+  group: config.openshift.io
+  names:
+    kind: ClusterOperator
+    listKind: ClusterOperatorList
+    plural: clusteroperators
+    singular: clusteroperator
+    shortNames:
+    - co
+  scope: Cluster
+  subresources:
+    status: {}
+  version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+---
+# XXX: This CRD will be removed before 4.0
```

Comment:
why is this `operatorstatus.openshift.io` needed?

Reply:
No idea, it's 1:1 copy paste of the CRD from https://github.com/openshift/cluster-version-operator/blob/master/install/0000_00_cluster-version-operator_01_clusteroperator.crd.yaml

Reply:
mm I don't think is needed, mao uses ConfigV1

Reply:
at the same time it does not hurt since it's used only in the testing. And again, it's 1:1 copy of the CRD pulled from CVO repo. I did not modify it to keep the original form of the file.

Reply:
the CVO was probably installing both for giving time to consumers to move to the new one. There's no reason to add the deprecated one here

Reply:
I removed the depricated CRD yet mentioning the pulled CRD is updated so it's documented what happened